### DISCUSSION TITLE
refresh activity in Elsa 3 design

### DISCUSSION THREAD

**Discussion by leostone68** [elsa-workflows/elsa-core#5667] [Design & tech]
Is it possible to refresh Activity in Else Studio Designer 3 ?

**Comment by glime-ai** [comment]
To provide an accurate answer, could you please clarify what you mean by "refresh Activity"? Are you asking if it's possible to update an existing activity within a workflow in the Elsa Studio Designer 3, or are you referring to refreshing the view of the designer itself to reflect changes made to activities?

This comment was generated by [Glime](https://glimelab.ai/?utm_source=dRbvTb7IhCSYiI&utm_medium=github).

**Reply by leostone68** [reply]
it's possible refreshing the view of activity and your input field  in designer?
